tutorials/katacoda/getting-started/index.json

```diff
@@ -0,0 +1,46 @@
+{
+  "title": "Getting started with Tekton",
+  "description": "Learn how to build a basic CI/CD workflow with Tekton",
+  "difficulty": "Beginner",
+  "time": "40 minutes",
+  "details": {
+    "steps": [
+      {
+        "title": "Core concepts",
+        "text": "concepts.md"
+      },
+      {
+        "title": "Setup",
+        "text": "setup.md"
+      },
+      {
+        "title": "Specifying Tekton tasks",
+        "text": "tasks.md"
+      },
+      {
+        "title": "Building tasks into a pipeline",
+        "text": "pipeline.md"
+      },
+      {
+        "title": "Adding the resources",
+        "text": "resources.md"
+      },
+      {
+        "title": "Running the pipeline",
+        "text": "run.md"
+      }
+    ],
+    "intro": {
+      "text": "intro.md"
+    },
+    "finish": {
+      "text": "finish.md"
+    }
+  },
+  "environment": {
+    "uilayout": "editor-terminal"
+  },
+  "backend": {
+    "imageid": "kubernetes"
```

Comment:
If we want to use the latest Pipelines version we'll need to update the image used. `kubernetes` provides a kubernetes v1.14 cluster, while Tekton Pipelines requires minimum v1.15 since the 0.11 release.

`"imageid": "kubernetes-cluster-running:1.18"`